packages/runtime/src/helpers/utils.ts

```diff
@@ -228,4 +228,7 @@ export const getRemotePatterns = (experimental: ExperimentalConfigWithLegacy, im
   }
   return []
 }
+
+export const isEnvSet = (envVar: string) => process.env[envVar] === 'true' || process.env[envVar] === '1'
```

Comment:
We really should be consistent about using this or destr. I'm not particularly bothered either way but it doesn't make sense to have both.